Blog.API/Blog.Dependencies/Dependencies.cs

```diff
@@ -67,4 +70,9 @@ private static void ConfigureMediatR(this IServiceCollection services)
     {
         services.AddMediatR(cfg => cfg.RegisterServicesFromAssemblyContaining<GetPostsHandler>());
     }
+
+    private static void ConfigureValidators(this IServiceCollection services)
+    {
+        services.AddValidatorsFromAssembly(typeof(CreateCommentRequestValidator).Assembly);
+    }
```

Comment:
I think it's better to have validation on Controller level. To do that you need to add
`services.AddFluentValidationAutoValidation();`
And if you do that, you'll need to remove validation from services and any other places.